deep_tempering/training_utils.py

```diff
@@ -150,7 +150,7 @@ def prepare_feed_tensors_and_values(self, training=True):
     for i in range(n_replicas):
       for hpname in hpnames:
         if not training and 'dropout_rate' in hpname:
-          value = 0.
+          value = 1.
```

Comment:
if this is not training the dropout rate should be 0. Why did you change it to 1?

Reply:
Older version of tf uses keep probability instead of dropout rate. But now this issue is managed in my custom model directly